runelite-client/src/main/java/net/runelite/client/ui/overlay/components/TooltipComponent.java

```diff
@@ -104,7 +104,7 @@ public Dimension render(Graphics2D graphics)
 					textComponent.setColor(nextColor);
 					String text = line.substring(begin, j);
 					textComponent.setText(text);
-					textComponent.setPosition(new Point(lineX, textY + (i + 1) * textHeight - textDescent));
+					textComponent.setPosition(new Point(lineX, textY + (i + 1) * metrics.getMaxAscent() + i * metrics.getMaxDescent()));
```

Comment:
You are positioning the components differently as the original tooltip height is calculated, use same logic in both places

Reply:
i think i got all your point except this one and the 2 similar ones below, can you clarify a bit more what's wrong here?

Reply:
I mean that textHeight should be dfeined using this logic here

Reply:
Unless I misunderstand you, I think that's not the case. Font is drawn from its base y position not from the top or bottom so textHeight is something different than this logic. This logic is for the Font base y.